configure.ac

```diff
@@ -2979,7 +2995,7 @@ PKG_PROG_PKG_CONFIG
 # Check for use of the system expat library
 AC_MSG_CHECKING(for --with-system-expat)
 AC_ARG_WITH(system_expat,
-            AS_HELP_STRING([--with-system-expat], [build pyexpat module using an installed expat library]),
+            AS_HELP_STRING([--with-system-expat], [build pyexpat module using an installed system expat library, see Doc/library/pyexpat.rst (default is no)]),
```

Comment:
Despite the 'system' in the name of the option, --with-system-expat doesn't mean expat needs to be installed on the system. It just means 'don't use the bundled one'. Leave out the word 'system' in the help text, let the doc ref explain it.